2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -39,7 +39,34 @@ const myBooks = [
 ];
 
 function createBookList(books) {
-  // TODO your code goes in here, return the ul element
+  const ulEl = document.createElement('ul');
```

Comment:
```suggestion
  const ulElement = document.createElement('ul');
```

Don't be afraid to have long variable names.. It's much more readable 😃